packages/block-library/src/post-time-to-read/edit.js

```diff
@@ -146,8 +141,7 @@ function PostTimeToReadEdit( { attributes, setAttributes, context } ) {
 		contentStructure,
 		blocks,
 		displayAsRange,
-		showTimeToRead,
-		showWordCount,
+		displayMode,
```

Comment:
Not from this PR, but as I was checking this file I saw that the `return parts.map( ( part, index ) => (` part above is not needed. We always `push` one item to `parts` array, so we should directly return the value in each `if` block that we now do `parts.push`